### DISCUSSION TITLE
How I can manage the notes sorting options?

### DISCUSSION THREAD

**Discussion by serrq** [aritra-tech/Notify#173] [General]
1.5.1 I cannot find a way to do this...

![1000093059](https://github.com/aritra-tech/Notify/assets/113377431/1513c749-5562-487c-bd06-69cfd48f62be)

**Comment by aritra-tech** [comment]
Actually, this feature was on the `develop` branch but when I merge it into `master` this feature is not there I don't know why this happened.
 I will look into it.

**Reply by aritra-tech** [reply]
Hey @serrq I have backtracked the PRs and found that somebody has removed that functionality.
Sorry for the inconvenience I will soon fix it up and create a new release.

**Reply by serrq** [reply]
Somebody? Do you haven't full control on approval process?
